products/cloudiot/terraform.yaml

```diff
@@ -0,0 +1,120 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+legacy_name: "cloudiot"
+overrides: !ruby/object:Overrides::ResourceOverrides
+  DeviceRegistry: !ruby/object:Overrides::Terraform::ResourceOverride
+    legacy_name: "google_cloudiot_registry"
+    import_format: ["{{project}}/locations/{{region}}/registries/{{name}}"]
+    id_format: "projects/{{project}}/locations/{{region}}/registries/{{name}}"
+    self_link: 'projects/{{project}}/locations/{{region}}/registries/{{name}}'
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      constants: templates/terraform/constants/cloudiot.go.erb
+      decoder: templates/terraform/decoders/cloudiot_device_registry.go.erb
+      encoder: templates/terraform/encoders/cloudiot_device_registry.go.erb
+      extra_schema_entry: templates/terraform/extra_schema_entry/cloudiot_device_registry.go.erb
+      pre_update: templates/terraform/pre_update/cloudiot_device_registry.go.erb
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloudiot_device_registry_basic"
+        primary_resource_id: "test-registry"
+        vars:
+          cloudiot_registry_name: "cloudiot-registry"
+        test_env_vars:
+          project: :PROJECT_NAME
+          region: :REGION
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloudiot_device_registry_single_event_notification_configs"
+        primary_resource_id: "test-registry"
+        vars:
+          cloudiot_registry_name: "cloudiot-registry"
+          cloudiot_device_telemetry_topic_name: "default-telemetry"
+        test_env_vars:
+          project: :PROJECT_NAME
+          region: :REGION
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloudiot_device_registry_full"
+        primary_resource_id: "test-registry"
+        vars:
+          cloudiot_registry_name: "cloudiot-registry"
+          cloudiot_device_status_topic_name: "default-devicestatus"
+          cloudiot_device_telemetry_topic_name: "default-telemetry"
+          cloudiot_additional_device_telemetry_topic_name: "additional-telemetr"
```

Comment:
```suggestion
          cloudiot_additional_device_telemetry_topic_name: "additional-telemetry"
```

Reply:
Fixed. This didn't cause any failure because it's just a name :)

good catch!